myadmin-webapps/myadmin-student-services-webapp/src/main/java/za/ac/unisa/myadmin/student/services/config/StudyMaterialServiceConfiguration.java

```diff
@@ -32,17 +37,18 @@ public StudyMaterialService studyMaterialService(){
 		return studyMaterialService;
 	}
 
-	@Bean("studyMaterialFilterActiveDecorator")
-	public StudyMaterialService studyMaterialFilterActiveDecorator(){
+	@Bean(name = "studyMaterialServiceImplRestEndPoint")
+	public Server studyFeeQuotationServiceImplRestEndPoint() {
```

Comment:
Copy paste error? studyFeeQuotationServiceImplRestEndPoint

Reply:
Fixed in follow up commit